apps/api/src/server/repositories/folder.repository.js

```diff
@@ -223,15 +223,16 @@ const mapFolderTemplateWithCaseId = (caseId, folder) => {
  * Returns an array of promises
  *
  * @param {number} caseId
+ * @param folders
```

Comment:
Should this be
```suggestion
 * @param {Folder[]} folders
```

or something similar?

Reply:
I'll use `FolderTemplate` - that is the type already assigned to `defaultCaseFolders` which is the default value for that arg